### PR TITLE
Now is possible to translate the extension author's name

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
Now is possible to translate the extension author.

Can also be translated contributors, even if they aren't yet shown in the ui.

(issue #9933, @datamweb)

### CODE REVIEW

**Comment by ingorichter** [comment]
@nicolo-ribaudo Thanks for the contribution. Please add a couple of testcases for the new functionality. This way we can test some edge cases and make sure we find out if something breaks.

File: src/extensibility/node/package-validator.js [hunk 1]
```diff
@@ -232,6 +232,29 @@ function validatePackageJSON(path, packageJSON, options, callback) {
                 }
             }
             
+            // normalize translated author and contributors
+            if (metadata["package-i18n"]) {
+                Object.keys(metadata["package-i18n"]).forEach(function (lang) {
+                    var packageLang = metadata["package-i18n"][lang];
+
+                    if (packageLang.author) {
```
Comment (peterflynn) [inline src/extensibility/node/package-validator.js]: This block (down to line 254) is duplicated with the code above, so please factor it into a reusable function.

File: src/extensibility/ExtensionManagerView.js [hunk 2]
```diff
@@ -250,6 +250,12 @@ define(function (require, exports, module) {
                             info.metadata[prop] = info.metadata["package-i18n"][locale][prop];
                         }
                     });
+                    if (info.metadata["package-i18n"][locale].hasOwnProperty("author")) {
+                        info.metadata.author = info.metadata["package-i18n"][locale].author;
+                    }
+                    if (info.metadata["package-i18n"][locale].hasOwnProperty("contributors")) {
+                        info.metadata.contributors = info.metadata["package-i18n"][locale].contributors;
+                    }
```
Comment (peterflynn) [inline src/extensibility/ExtensionManagerView.js]: I'm not sure I understand why these need special treatment, instead of just adding `"author"` and `"contributors"` to the array above.

Comment (peterflynn) [inline src/extensibility/ExtensionManagerView.js]: Btw, we may want to add a comment here explaining that `"author"` is safe for localization because it's just a "friendly name" and the GitHub user id is stored in a separate `"owner"` field (which will never be localized).

**Comment by nicolo-ribaudo** [comment]
@peterflynn I've cleaned up the code.

@ingorichter Are there already some translation test that I can extend?

**Comment by ficristo** [comment]
@nicolo-ribaudo can you solve the merge conflicts?
I think this still miss the comment requested from @peterflynn

**Comment by nicolo-ribaudo** [comment]
I'm sorry but I don't use Adobe Brackets anymore. If anyone is interested in cherry-picking my commits and make the needed changes, feel free to do that.
